src/java.base/share/classes/java/lang/runtime/SwitchBootstraps.java

```diff
@@ -48,6 +49,9 @@
 import java.lang.classfile.ClassFile;
 import java.lang.classfile.Label;
 import java.lang.classfile.instruction.SwitchCase;
+
+import jdk.internal.constant.MethodTypeDescImpl;
```

Comment:
Nit - this import seems to be unused, and neither seem to be the changes to `MethodTypeDescImpl`. Is there some use missing? (OTOH, I like the `StaticHolders` for the stuff that relates to the enum-switch special case, for the time being.)

Reply:
Oops, some leftovers from the optimizations I reverted. Fixed.